en/docs/deploy/vm-based-deployment/centralized-deployment.md

```diff
@@ -4,86 +4,86 @@ Managing a large number of BI artifacts across different environments can become
 
 Centralized deployment typically involves **two repositories**:
 
-### Source Repository (CI)
+## Source Repository (CI)
```

Comment:
These should be in sentence case, Only the Level 1 headings should be in title case. Check and fix all the places.